app/config/security.yml

```diff
@@ -37,11 +37,8 @@ security:
             logout_on_user_change: true
             logout:
                 path: /logout
-                target: "%logout_redirect_url%"
+                target: logout_redirect
```

Comment:
This feels like a over-complicated solution. 

1. We are not actually logging out, so using the built in logout feature might not be that useful. When you return to SPD after visiting the page. You have not lost your SSO.
2. Maybe just create a single logout action (like you did) and return a redirect response there.

Reply:
Good points.  I simplified it and removed the logout logic in the security.yml (as it'd conflict otherwise).  Nice'n simple that way 💯